pkg/apiserver/archive/experiment.go

```diff
@@ -81,3 +84,74 @@ func (s *Service) listExperiments(c *gin.Context) {
 
 	c.JSON(http.StatusOK, data)
 }
+
+// @Summary Get the details of chaos experiment.
+// @Description Get the details of chaos experiment.
+// @Tags archives
+// @Produce json
+// @Param namespace query string false "namespace"
+// @Param name query string false "name"
+// @Param kind query string false "kind" Enums(PodChaos, IoChaos, NetworkChaos, TimeChaos, KernelChaos, StressChaos)
+// @Param uid query string false "uid"
+// @Success 200 {array} core.ArchiveExperiment
+// @Router /api/archives/detailSearch [get]
+// @Failure 500 {object} utils.APIError
+func (s *Service) experimentDetailSearch(c *gin.Context) {
```

Comment:
If we define this API as a search API, I think `kind`, `name`, and `ns` may be empty and we should return a list of the experiment instead of the single object.